frontend/src/metabase/admin/permissions/selectors/data-permissions/groups.ts

```diff
@@ -1,16 +1,26 @@
 import { createSelector } from "@reduxjs/toolkit";
 import _ from "underscore";
+import { assoc } from "icepick";
 
 import Groups from "metabase/entities/groups";
 import { Group } from "metabase-types/api";
-import { isAdminGroup, isDefaultGroup } from "metabase/lib/groups";
+import {
+  getGroupNameLocalized,
+  isAdminGroup,
+  isDefaultGroup,
+} from "metabase/lib/groups";
 
 const isPinnedGroup = (group: Group) =>
   isAdminGroup(group) || isDefaultGroup(group);
 
 export const getOrderedGroups = createSelector(
   Groups.selectors.getList,
-  (groups: Group[]) => _.partition(groups, isPinnedGroup),
+  (groups: Group[]) => {
+    const translatedGroups = groups.map(group =>
```

Comment:
Appears in
1. Admin > Permissions > Data > Group
    ![Screenshot 2023-08-09 at 6 14 29 PM](https://github.com/metabase/metabase/assets/1937582/b4795cc9-2c28-41b9-ab2b-0aa33c7cdaf5)

And a few other places that I'm too tired to track right now. But you can try to find the usage of `getOrderedGroups` to see which components this selector will be used on.